packages/web-components/src/components/ic-breadcrumb/ic-breadcrumb.spec.ts

```diff
@@ -119,4 +119,15 @@ describe("ic-breadcrumb", () => {
       document.querySelector("ic-breadcrumb").classList.contains("current-page")
     ).toBeTruthy;
   });
+
+  it("should call 'setFocus' when breadcrumb is focused", async () => {
+    const page = await newSpecPage({
+      components: [Breadcrumb],
+      html: `<ic-breadcrumb></ic-breadcrumb>`,
+    });
+
+    await page.rootInstance.focus;
```

Comment:
I don't think this line is doing anything in terms of testing the focus (in any of the focus tests). I think the actual execution of the setFocus function is coming from the `await page.rootInstance.setFocus` on the next line

Reply:
I would imagine that in an 'expect' statement, once you have called setFocus(), you would check that document.activeElement is the correct HTML element (e.g. the 'ic-link' here) :)

Reply:
I would imagine that in an 'expect' statement, once you have called setFocus(), you would check that document.activeElement is the correct HTML element (e.g. the 'ic-link' here) :)

Reply:
This is how the tests are implemented for the existing setFocus methods, should I go through and update all of those?

Reply:
This is how the existing test for button's setFocus has been done. Not sure if the above is possible, it seems accessing the document.activeElement is only possible in e2e tests like `const activeElement = await page.evaluate(() => document.activeElement.id);` but testing there wouldn't make the coverage go up.